CHANGELOG.md

```diff
@@ -1,6 +1,104 @@
 
+## [3.0.0-rc.1] 2019-05-09
+
+### SQL Optimizer
+* Improve the accuracy of cost estimates by using the order correlation between columns; introduce a new heuristic parameter `tidb_opt_correlation_exp_factor` to control the preference for index scans; when the value of the parameter is larger, index scans are more preferred [#9839](https://github.com/pingcap/tidb/pull/9839)
+* Match more prefix columns of the indexes when extracting access conditions of composite indexes if there are relevant columns in the filter [#10053](https://github.com/pingcap/tidb/pull/10053)
+* Use the dynamic programming algorithm to specify the execution order of join operations when the number of tables participating in the join is less than the value of `tidb_opt_join_reorder_threshold`; the default value of this parameter is 0, so the greedy algorithm is used by default [#8816](https://github.com/pingcap/tidb/pull/8816)
+* Match more prefix columns of the indexes in the inner tables that build the index join when using composite indexes as the access conditions [#8471](https://github.com/pingcap/tidb/pull/8471)
+* Improve the accuracy of row count estimation for single-column indexes with NULL values [#9474](https://github.com/pingcap/tidb/pull/9474)
+* Specially handle `GROUP_CONCAT` when eliminating aggregate functions during the logical optimization phase to prevent an incorrect result [#9967](https://github.com/pingcap/tidb/pull/9967)
+* Properly push the filter down to child nodes of the join operator if the filter is a constant [#9848](https://github.com/pingcap/tidb/pull/9848)
+* Specially handle some functions such as `RAND()` when pruning columns during the logical optimization phase to prevent a result with an incompatibility with MySQL [# 10064](https://github.com/pingcap/tidb/pull/10064)
+* Support `FAST ANALYZE`, which speeds up statistics collection by sampling the region instead of scanning the entire region. This feature is disabled by default. To enable it, set the variable `tidb_enable_fast_analyze` to true [#10258](https://github.com/pingcap/tidb/pull/10258)
+* Support SQL Plan Management, which ensures execution stability by performing execution plan binding on SQL statements. This feature is currently in beta and only supports using bound execution plans for SELECT statements. It is not recommended to use it in the production environment. [#10284](https://github.com/pingcap/tidb/pull/10284)
+### Execution Engine
+* Support tracing and controlling memory usage in three operators - `TableReader`, `IndexReader` and `IndexLookupReader` [#10003](https://github.com/pingcap/tidb/pull/10003)
+* Support showing more information about coprocessor tasks in the slow log such as the number of tasks in coprocessor, the average/longest/90% of execution/waiting time and the longest execution/waiting time of the TiKV address [#10165](https://github.com/pingcap/tidb/pull/10165)
```

Comment:
`the longest execution/waiting time of the TiKV address` is not correct.

It's `the addresses of the TiKVs which take the longest execution time or waiting time`

Reply:
PTAL @lilin90

Reply:
@yikeke Please take a look. Thanks!